cluster/loadbalance/consistent_hash.go

```diff
@@ -53,16 +53,16 @@ func init() {
 	extension.SetLoadbalance(ConsistentHash, NewConsistentHashLoadBalance)
 }
 
-// ConsistentHashLoadBalance ...
+// ConsistentHashLoadBalance Implementation of load balancing: using consistent hashing
```

Comment:
See https://golang.org/doc/effective_go.html#commentary, after function name, the first character of word don't need to be capitalized.